tests/test-account-alternates/main.tf

```
@@ -0,0 +1,47 @@
+module "test-account-alternates" {
+  source = "../.."
+
+  # account_alternate_contacts = {
+  #   billing = local.result
+  #   operations = local.result
+  #   security = local.result
+  # }
```

Let's use this approach, re-using the local value... then delete the not-used code and comments.

```suggestion
  account_alternate_contacts = {
    billing    = local.result
    operations = local.result
    security   = local.result
  }
```

---

- updated locals {}
- updated workflows
- updated release
- updated changelog 

```terraform-aws-tardigrade-account-alternate-contacts on  initialdeploy [!?] via 🐳 desktop-linux via 💠 default on ☁️  aws (us-east-1) 
✦2 ❯ make test PYTEST_ARGS="-v --nomock"
[bootstrap]: Using external tardigrade-ci, skipping auto-init
[pytest//Volumes/Development/tardigrade-ci/tests/terraform_pytest] Starting Python tests found under the directory "/Volumes/Development/tardigrade-ci/tests/terraform_pytest"
pytest /Volumes/Development/tardigrade-ci/tests/terraform_pytest -v --nomock 
=========================================================================================== test session starts ============================================================================================
platform darwin -- Python 3.10.9, pytest-7.2.1, pluggy-1.0.0 -- /Users/johnricords/.local/opt/python@3.10/bin/python3.10
cachedir: .pytest_cache
rootdir: /Volumes/Development
collected 1 item                                                                                                                                                                                           

../tardigrade-ci/tests/terraform_pytest/test_terraform_install.py::test_modules[test-account-alternates] PASSED                                                                                      [100%]

============================================================================================ 1 passed in 21.04s ============================================================================================
[pytest//Volumes/Development/tardigrade-ci/tests/terraform_pytest]: Tests executed!```